src/openforms/submissions/tests/test_views.py

```diff
@@ -54,9 +55,17 @@ def test_successfully_submit_form(self):
         form["code"] = submission.public_registration_reference
         submission_response = form.submit()
 
+        expected_redirect_url = furl("http://url-to-form.nl/")
+        expected_redirect_url.add(
+            {
+                "_of_action": "cosign",
+                "_of_action_params": f"submission_uuid={submission.uuid}",
+            }
+        )
```

Comment:
Maybe a helper assertion with this signature would be nice:

```py
def assertRedirectsToFrontend(
    self,
    response,
    frontend_base_url: str,  # *may* contain query string parameters
    action: Literal[...],  # defined in openforms.frontend
    action_params: dict[str, str],
    **kwargs,
) -> None:
```

the assertion helper then performs the encoding itself (and you can write unit tests for the assertion itself too) and building the final URL as a string, and finally calls the underlying `self.assertRedirects(...)`